### DISCUSSION TITLE
Timeline for 4.0 inference submission

### DISCUSSION THREAD

**Discussion by mrasquinha-g** [mlcommons/inference#1503] [Ideas]
Please review timeline for 4.0 https://docs.google.com/spreadsheets/d/1p09zafJIsBRgKqj5ULuOpaZLfLwCvw2n87mDFOTNTcM/edit?usp=sharing

Comment access is provided for inference@

**Comment by psyhtest** [comment]
I'd propose:
- Random seeds to be released on week -5 (ending on Fri 19 Jan) - Tue 16 Jan?
- Submission UI to open for testing on week -4 (ending on Fri 26 Jan) - Thu 25 Jan?
- Submission UI to open for submissions on week -3 (ending on Fri 2 Feb) - Thu 1 Feb?
